onboarding/config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  #post '/customer/create' => 'customer#create', as: :create_customer
+  #get '/customer/find_all' => 'customer#find_all', as: :find_all
+  resources :customers do
+    get 'find_all'
```

Comment:
I think this won't give you the expected path. Compare the routes using `bundle exec rails routes` before and after this change so you will see the difference. Also, take a look at https://guides.rubyonrails.org/routing.html#adding-collection-routes